scripts/run_tests_in_ci.js

```diff
@@ -52,10 +57,14 @@ const argv = yargs.options({
 
     await testProcess;
     console.log('Success: ' + name);
+    appendFileSync(LOGFILE, stdout + '\n' + stderr, { encoding: 'utf8' });
```

Comment:
Will there be any race condition ?

Reply:
There will absolutely be race conditions. I changed it to write multiple files.